### PR TITLE
MAS Bug Fix finally complete

### PR DESCRIPTION
**Pull request by mas1850**
I finally managed to get the project to run and fixed my chosen bug

### CODE REVIEW

File: _posts/2024-04-21-schneider-joycon.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+layout: post
+# If your post title is longer or more complicated
+# than can be represented in the filename, uncomment the following line
+# and specify a custom title
+# title:  "Sample Blog Post"
+
+
+# Uncomment only one of the below categories
+categories:
+Bug Fix
```
Comment (MoralCode) [inline _posts/2024-04-21-schneider-joycon.md:11]:
```suggestion
- Bug Fix
```
Small formatting issue thats causing Jekyll to not like this file

File: _posts/2024-04-21-schneider-joycon.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+![original retail_colors.xml code](../assets/2024-04-21-schneider/code-snippet-1.png)
+
+For my first draft, I made a few changes with some inconsistencies to learn the behavior of the design. My first assumption was to add more RC tags to the RetailColors and RetailColorsButtons tags. I added the four new colors to each parent tag and adjusted the numbers to increase as expected. Wondering if the preset number was the determining factor of the color's position, I changed the Pastel Purple color preset from 11 to 26.
+
+![first edit to retail_colors.xml code](../assets/2024-04-21-schneider/code-snippet-2.png)
+
+Here, I have many issues trying to debug the Visual Studio solution. I received a multitude of error messages, including issues with Windows SDK and .NET version differences. After meeting with Adrian, I was eventually able to launch the solution on my laptop. This is when I noticed no change within the program.
+
+![error pop-up message after editing retail_colors.xml](../assets/2024-04-21-schneider/error-message.png)
```
Comment (MoralCode) [inline _posts/2024-04-21-schneider-joycon.md:54]: a couple of your image file extensions (such as this one, but also a few others throughout your post) are saved as .jpg but referenced as .png and not showing up

```suggestion
![error pop-up message after editing retail_colors.xml](../assets/2024-04-21-schneider/error-message.jpg)
```

**Review by MoralCode — COMMENTED** [review]
Looks good overall! Just a couple minor tweaks

File: _posts/2024-04-21-schneider-joycon.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+After fiddling with the UI some more, I found a label titled "Xenoblade Grips". Knowing that the word Xenoblade was unlikely to be used frequently, I searched the project for the word and found the correct file, named "frmJoyConColorPicker.cs". Using the foundations for color slots 17-20 and 25-28 left in the C# source code file, I added the labels and RGB codes for the eight new pastel colors.
+
+![edit to frmJoyConColorPicker.cs code](../assets/2024-04-15-schneider/code-snippet-3.jpg)
```
Comment (MoralCode) [inline _posts/2024-04-21-schneider-joycon.md:58]: looks like there was just one image that slipped through
```suggestion
![edit to frmJoyConColorPicker.cs code](../assets/2024-04-21-schneider/code-snippet-3.jpg)
```

**Comment by mas1850** [comment]
Done

File: _posts/2024-04-21-schneider-joycon.md (hunk 2 above)
Comment (MoralCode) [reply _posts/2024-04-21-schneider-joycon.md:54]: @mas1850 It still seems to be this specific image that isnt working

**Pull request merged**